projects/plugins/crm/includes/ZeroBSCRM.SystemChecks.php

```diff
@@ -25,27 +25,69 @@
   Generic System Check Wrapper/Helper funcs
    ====================================================== */
 
-	#} Can use to check php feat is properly installed :)
-	#} zeroBSCRM_checkSystemFeat('zlib')
-	function zeroBSCRM_checkSystemFeat($key='',$withInfo=false){
-
-		$featList = array('zlib','dompdf','mb_internal_encoding','pdffonts','curl','phpver','wordpressver','locale','assetdir','executiontime','memorylimit','postmaxsize','uploadmaxfilesize','wpuploadmaxfilesize','dbver','dalver','corever','local','localtime','serverdefaulttime','sqlrights','devmode','permalinks','mysql','innodb', 'fontinstalled', 'encryptionmethod' );
+/**
+ * Check if a PHP systems/library is correctly installed.
+ *
+ * @param string $key The feature we want to know the status off.
+ * @param bool   $with_info Weather we should return a message or not.
+ *
+ * @return array|false
+ */
+function zeroBSCRM_checkSystemFeat( $key = '', $with_info = false ) {
```

Comment:
Renaming this one is a can of worms because it's used in other paid functions as well and, linting wise, this will force me to update the places its being used as well which then triggers other linting issues that are outside the scope of this PR.

Reply:
As I mentioned, I don't mind ignoring that one here. You have several options:

* Put `// phpcs:ignore WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid` at the end of this line.
* Put `// phpcs:disable WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid` near the top of the file, maybe with additional commentary as to why (e.g. permanently ignoring, or "TODO should be fixed later").
* Add something like
  ```
  <!-- We don't like these parts of the WordPress Coding Standards for this plugin. -->
  <rule ref="WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid">
  	<severity>0</severity>
  </rule>
  ```
  to `projects/plugins/crm/.phpcs.dir.xml` to ignore it for the whole plugin.